crates/bevy_time/src/lib.rs

```diff
@@ -214,8 +222,10 @@ mod tests {
         let time_step = fixed_update_timestep / 2 + Duration::from_millis(1);
 
         let mut app = App::new();
+        #[cfg(feature = "fixed_update")]
```

Comment:
I changed this to make sure the tests only exist for the right feature since they all need to have `fixed_time` enabled to work in the first place so testing without the feature enabled should run no tests